recipe/meta.yaml

```
@@ -29,6 +29,7 @@ requirements:
     - click >=5.1
     - filelock
     - requests >=2
+    - packaging
```

```suggestion
    - packaging
    - requests >=2
```
Very minor: It is easier to review PRs when the dependencies are in the same order than the dependencies upstream.